arch/mips/src/pic32mx/pic32mx_ethernet.c

```diff
@@ -215,7 +215,7 @@
  * header
  */
 
-#define BUF ((struct eth_hdr_s *)priv->pd_dev.d_buf)
+#define BUF ((FAR struct eth_hdr_s *)priv->pd_dev.d_buf)
```

Comment:
remove

Reply:
Currently `FAR` is heavily used in arch/mips files

Reply:
Ok, let's keep it.